spec/requests/api/service_dialogs_spec.rb

```diff
@@ -62,6 +62,39 @@
 
       expect_result_to_have_only_keys(%w(href id label))
     end
+
+    context 'Delete Service Dialogs' do
+      it 'DELETE /api/service_dialogs/:id' do
+        dialog = FactoryGirl.create(:dialog)
+        api_basic_authorize collection_action_identifier(:service_dialogs, :delete)
+
+        expect do
+          run_delete(service_dialogs_url(dialog.id))
+        end.to change(Dialog, :count).by(-1)
+        expect(response).to have_http_status(:no_content)
+      end
+
+      it 'POST /api/service_dialogs deletes single service dialog' do
+        dialog = FactoryGirl.create(:dialog)
+        api_basic_authorize collection_action_identifier(:service_dialogs, :delete)
+
+        expect do
+          run_post(service_dialogs_url, 'action' => 'delete', 'resources' => [{ 'id' => dialog.id }])
+        end.to change(Dialog, :count).by(-1)
+        expect(response).to have_http_status(:ok)
+      end
```

Comment:
this test here can be changed to test POST /api/service_dialogs/:id deletes single service dialog since we already the multiple service_dialogs below (or add another test).  i.e. run_post(service_dialogs_url(dialog.id), 'action' => 'delete')  Thanks.